src/redux/reducers/session.js

```diff
@@ -0,0 +1,18 @@
+const INITIAL_STATE = {
+  authUser: null
+};
+
+const applySetAuthUser = (state, action) => ({
+  ...state,
```

Comment:
For instance,  you can not figure out what is in "state" because the definition of "state" is at somewhere that can not be seen here.